### ISSUE TITLE
[Floors] Floor themes in Agony!

### ISSUE THREAD

**Issue by ProbablyTez** [ProbablyTez/Agony#392]
- Basement 
   - Puzzle rooms with little to no enemies.
   - Sackdudes, Flies and Poop.
  -Note: Sackdudes in Basement are the Gaper model and become one upon death.

- Cellar 
    - Spiders, Cobwebs, the hanging nests( #450 )
    - Barrels (will link to idea card later)

- Burning Basement
    - More fire alts, specifically mostly for the basic common basement enemies.

- Caves
    - Underground creatures ; maggots, insects, slightly decayed corpses

- Catacomb
     - Mushrooms and toxic gas
     - Spores : #399 

- Flooded Caves
     - Bloated corpses, low hp but highly dangerous

- Depths
     -Dying enemies. Not already dead, not undead, dying. 
      -Knights. Lotsa Knights.

- Dank Depths 
   -Being made out of, engulfed, or stuck in gooey sticky Tar.

- Necropolis
     - Masks n Hearts, like how Depths has it's knights.
     - Skeletones and generally undead.

- Womb
     - Skinless things, meat, and birth related stuff.

- Utero
     - Globins, Bones/Guts(Not the enemy, organs).

- Scarred Womb
    - Basically the gorier shit or more disturbing stuff. Poorly attached flesh(like meatballs) , severe birth defects, stuff like Fester(A botfly maggot feasting off of the floor).

- Cathedral 
    - A demented chapel.

- Sheol
    - Home of Demons.

- Chest 
    - smth idk

- Dark Room
    - Religious things, twisted into Demons. Basically like Lamb is a demented Jesus.

- Void
    - Merged enemies (will link idea card) and Delusions.

**Comment by Floofball** [comment]
A couple of suggestions if I may ...

For the Catacombs, rather than a Mushroom theme (I think it would mesh much better with the Caves), why not have a 'Bricked' kind of theme to it? Here's my line of thought: think of the Cask of Amontillado when Montresor traps Fortunato in, well, what ARE essentially catacombs, by bricking a wall over so he can't escape, and then take that idea of after so many years of being trapped in there he's just a skeleton. That way, the Catacombs stay similar in theme (skeletons), but will have a bit more of a 'forgotten' vibe to it, as if this is the first times these corpses have even been awake since their death. 

I'm imagining props that are literally just giant piles of bones and skulls, and other enemies, mummified or skeletal, that can rise up from the ground, pits, or even burst through walls.

As for the Depths, why not a 'Dungeon' (or 'Spike and Chain') theme? Have enemies that shoot out chain whips when the player is in front of them, or 'Black Knights' that smash into walls to get to you. Maybe even have some enemies with balls and chains! According to NotYourSaggitarius, this is kind of what the Dank Depths were intended to be originally, and I personally think it is a much better idea than the alt we got in AB, so seeing a sort of spiritual resurrection for that idea but without replacing the Dank Depths would be great, especially because I have my own little idea for that floor.

The Dank Depths, or maybe now better known as 'The Halls', will be a COMPLETE rework of the boring vanilla's concept. Sticky tar is just kind of, lacking ... And personally, I feel as if another think that this game lacks is demons, and lots of em! This floor would be completely retooled visually to have it's center focus on the occult and demonic covens. Aside from Sheol, you really don't see the demon enemies that much, so having them appear here without bosses like the Fallen, as well as some original ones (imagine enemies wearing Ceremonial Robes and summoning (WEAKENED) Imps) and bosses would make this floor a treat for players who were never really that impressed with the Dank Depths (like me).

Hwoof, that was a lot of typing. Respond as soon as you can! I'd love to here your thoughts and, if my ideas get approved, I'll take on the brunt of the work for it as recompense.

**Comment by Anonymous-Koala** [comment]
Issue with giving Catacombs that theme, Floofball, is that Depths and Necropolis are the ones with the focus on death and skeletons.  Although I like the idea of the giant piles of bones.

The Depths theme, while good is again, problematic, as for the most part we're not adding a new theme, but just working off of what the floor already has for a theme. 

Demons are better fit for Sheol, though. And the thing with why it's good demonic enemies aren't all that many outside of there is because it gives Sheol a stronger personality, and that's what we want to do to all the floors. Give them unique enemies to make them feel like more unique floors. I do like the idea of an enemy wearing Ceremonial Robes though.

Dank Depths' tar theme is very much a more decayed one. You can do a lot of interesting things with it once you open your mind a bit. Using Tar being flameable in order to have a boss that mixes the two(I could see it being twins), a boss or enemies that sink into the floor before re-meterializing, etc.

**Comment by Floofball** [comment]
@Anonymous-Koala Hmm, you've got some good points there, but there are others I disagree with you on.

In particular, you say that the Depths has a death and decay theme before switching and saying that it has the dungeon theme. To give you an example of what I mean, the user I mentioned previously included this link in their post about the floor concept: https://puu.sh/qDw8O.jpg In the Depths you have some corpses but that's really about it: there just isn't that much there that you can point to and say "Oh, that's the Depths" about anything BESIDES its appearance. Having enemies like the ones I mentioned and through the link I posted just adds identity where it is lacking, at least that's how I see it.

As for Sheol, I heavily disagree on it having personality. Now, this may just be me, but Sheol has always felt lackluster and rushed, even moreso than the Cathedral. It's backdrop is the uninspiring one you see draped in Devil Deals and Trap Rooms all over the game, and most of the enemies there by that point don't even really pose a legitimate threat. Honestly, I'd argue the Womb is a harder floor. And despite the fact that it has a lot of demons, it still feels empty, like there's just something missing. Also, the Halls wouldn't really be a ripoff of Sheol rather than a worship floor. To simplify it into one sentence: The Halls are the cathedral of demons, and Sheol is the HOME of the demons. Now, to be fair, I wasn't very explanatory in my post, so I can forgive any misunderstandings there. 

The Halls, rather than being these dark, blackened floors of stone would instead be a demonic alt to the cathedral, minus the windows. Reddish tints/fog would cover everything, and fires would be replaced with 'Candle' props (ranging from blue, red, purple, even black), with the floor being a sort of dusty, blood stained floor. Enemies that summon demons but are weak themselves like the ones I described, other Occult members that circle around you to try and inflict the Black Powder effect etc. would populate the area more so than ACTUAL demons. Yeah, there'd be some, definitely more than your typical Chapter 3, but would really be home to Satanic worshippers rather than the demons themselves.

As for the tar theme being distinct, I guess I should've explained further. Honestly, I probably should have read over my post to make sure it had more info, but w/e. What I mean to say is that, as it is now, it is INCREDIBLY lackluster, and it doesn't really fit the 3rd Chapter either. Now, one way to solve this and keep the tar/slowing/sticky theme was if it was moved to the Scarred Womb, which would just be FLESHED out (haha get it? I'll pick up Transendence now, sorry ...) to have patches of scabs and blood clots all over the place.

And finally, I agree with you on the Catacombs thing. Rather than have skeletons, maybe just mummified enemies? Kind of like exclusive 'Ragman' esque alts and such. Also, maybe new Grimaces or even ghosts?

**Comment by Anonymous-Koala** [comment]
Yup I've seen the concept. That's actually a fair enough point. What we offered for the theme is mostly that Depths is rotting and dying, Necropolis is after that, Dank Depths is Tar. Might work, I wonder what Tez has to say.

That's a better point. I like the idea of a worship ground, but I'm not sure where would it fit. Also I agree about Sheol enemies not being half as difficult as they should be, but then again, neither are Cathedral enemies. Both of these floors' difficulty levels probably will be tweaked either way though.

I do love the idea of Black Powder being applied to an enemy.

Of course it's lackluster as it is now, all the AB alts are. You have Basement with random fires and enemies which are on fire but don't do anything special with that property, Caves with water that does nothing and has tanky recolored enemies, Depths which has slightly blacker walls and one or two "tar" enemies which really are just fancy Gishes, and Womb with skin, which doesn't do anything at all with that. 

Not sure about Catacombs, although mummified enemies I do think would fit in one of the other floors, whichever one it may end up being. New grimaces are a must at some point, there's a lot of fun ideas to explore with them.

**Comment by Floofball** [comment]
I still think that the Halls (maybe the Grounds or something, I have no idea what would be a better name) would still fit fine as a replacement for the Dank Depths. Just move that sticky tar theme to the Scarred Womb. As I said, have enemies that are literally just scabs and blood clots, maybe malformed, sticky lumps that pop out of blood pools, etc. Also stationary enemies that replace Nerve Ending that don't do contact damage but do slow you down, kind of like living cobwebs. Fits better with the Scarred Womb's theme and allows for other ideas in place of the Dank Depths.

**Comment by Anonymous-Koala** [comment]
That could work, about Scarred Womb. Dank Depths I suppose we'll have to see with Tez. Although I like the idea of adding chains to make Depths more unique, as the difference between it and Necropolis may not be too noticeable otherwise.

**Comment by ProbablyTez** [comment]
@Floofball I agree that the Dank Depths have a shitty theme. However, Agony never replaces or removes anything, so we have to improve on the theme by additional content only. Think of Tar like the most decayed state of death. Literally organic puddle + bones. There's great potential there. 

Depths' theme is pretty tough. Dungeon-y theme seems pretty obvious, though I feel like it could get wrong very very fast. Non organic stuff is very rare in Isaac, and a dungeon floor could very easily feel liek it doesnt fit.

Someone talked about an enemy having black powder, its pretty great, you shoudl post that as an idea. A lil note about this, have a look at the wizard boss in the rogue like mini game in Night In The Woods, the one which only moves in a pentagram pattern, I think it'd work great in Isaac
